parcels/examples/example_peninsula.py

```diff
@@ -140,6 +141,7 @@ def fieldsetfile(mesh):
 @pytest.mark.parametrize('mesh', ['flat', 'spherical'])
 def test_peninsula_file(mode, mesh):
     """Open fieldset files and execute"""
+    gc.collect()
```

Comment:
Again, why exactly is this `gc` necessary?

Reply:
See above